src/utility/watchdog/Watchdog.cpp

```diff
@@ -154,4 +173,13 @@ void watchdog_reset()
   mbed_watchdog_reset();
 #endif
 }
+
+void watchdog_enable_network_feed(bool use_ethernet)
```

Comment:
In how far is `watchdog_enable_network_feed` different from `watchdog_enable`. What are the advantages of one over the other? I don't think I get it.

Reply:
`watchdog_enable` enables the mcu watchdog
`watchdog_enable_network_feed` enables watchdog kiks/resets during network operations. This was introduced with:
* https://github.com/arduino-libraries/ArduinoIoTCloud/pull/260
* https://github.com/arduino-libraries/WiFiNINA/pull/186/
* https://github.com/arduino/ArduinoCore-mbed/pull/258